.github/workflows/openshift-ci/scripts/install_tini.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
what is tini for?

Reply:
it is a simple bash equivalent for Alpine as far as I can tell: tbh I used the bats-core docker image as reference here